src/backend/serial/u32/constants.rs

```diff
@@ -234,11 +234,12 @@ pub const EIGHT_TORSION_INNER_DOC_HIDDEN: [EdwardsPoint; 8] = [
 ];
 
 /// Table containing precomputed multiples of the Ed25519 basepoint \\(B = (x, 4/5)\\).
-pub const ED25519_BASEPOINT_TABLE: EdwardsBasepointTable = ED25519_BASEPOINT_TABLE_INNER_DOC_HIDDEN;
+pub static ED25519_BASEPOINT_TABLE: &'static EdwardsBasepointTable =
```

Comment:
I think this can be `const` considering that it points to an actual static allocation (`ED25519_BASEPOINT_TABLE_INNER_DOC_HIDDEN`)

Reply:
Nope

```
error[E0013]: constants cannot refer to statics
   --> src/backend/serial/u64/constants.rs:325:6
    |
325 |     &ED25519_BASEPOINT_TABLE_INNER_DOC_HIDDEN;
    |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```